ios/ParentBuddyAppXk-Info.plist

```diff
@@ -46,6 +46,16 @@
 				<string>bebbo</string>
 			</array>
 		</dict>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
+			<string>Bundle ID</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>org.unicef.kosovo.foleja</string>
```

Comment:
Do we add similar changes in the ParentBuddyAppXkDev-Info.plist as well?

Reply:
No need to change in dev build plist file as this change is for prod build only.